web/app/scripts/src/services/fields-config.es.js

```diff
@@ -6,6 +6,7 @@ export const moduleProperties = {
 };
 
 const module = angular.module(moduleProperties.moduleName, [
+    'incuna-surveys-fields.templates',
```

Comment:
I'm not sure that this is required, as the project creates a templates modules. @grahamgilchrist ?